source/extensions/extensions_build_config.bzl

```diff
@@ -270,6 +270,15 @@ EXTENSIONS = {
     #
 
     "envoy.http.stateful_header_formatters.preserve_case":       "//source/extensions/http/header_formatters/preserve_case:preserve_case_formatter",
+
+    #
+    # Original IP detection
+    #
+
+    # Disabled by default
+    "envoy.http.original_ip_detection.custom_header":        "//source/extensions/http/original_ip_detection/custom_header:config",
+    "envoy.http.original_ip_detection.xff":                  "//source/extensions/http/original_ip_detection/xff:config",
```

Comment:
Can we make it clear somehow that xxff isn't disabled by default?  I'd be inclined to split into 2 sections so have this above the Disabled comment, but you could also just add a commet in between to make it clear it's not a block of disabled filters i that makes sense?

Reply:
Not a thing anymore, brought it back to the build.